packages/next/src/routes/rest/files/getFile.ts

```diff
@@ -55,12 +56,18 @@ export const getFile = async ({ collection, filename, req }: Args): Promise<Resp
     const filePath = path.resolve(`${fileDir}/${filename}`)
 
     const stats = await fsPromises.stat(filePath)
+
     const data = streamFile(filePath)
 
+    const headers = new Headers({
+      'content-length': stats.size + '',
+    })
+
+    const contentType = mime.contentType(path.extname(filePath))
+    if (contentType) headers.set('content-type', contentType)
```

Comment:
I would rather not install another library for mime-types. Could we utilize `file-type` that is installed in payload. We could export a function from Payload that does this using that library and then import that function into the Next package here. I am thinking something simple like this:

```ts
// payload/utilities/getFileDetails

import fileType from 'file-type'

export const getFileType = async ({ filePath }) => {
  return fileType.fromFile(filePath)
}

// OR we could just export the function `fileType` from within payload
// Not sure which would be preferre

export const getFileType = fileType
```

Reply:
Could it be more more meaningful if we would place this to payload/uploads instead if payload/utilities?

Reply:
![image](https://github.com/payloadcms/payload/assets/64744993/2a16c939-c311-45f9-8eee-33bbe58c41f8)
Yea it seems like front end imports utillities and file-type uses server side `fs` package

Reply:
@r1tsuu **oic**, ok lets move it to `exports/node.ts` then 👍

Reply:
> @r1tsuu **oic**, ok lets move it to `exports/node.ts` then 👍

oh i commited, i wil

> @r1tsuu **oic**, ok lets move it to `exports/node.ts` then 👍

done, i also removed check from `mime` in headers because i think it should be fine